pkg/server/endpoints/endpoints.go

```diff
@@ -124,6 +126,8 @@ func (e *Endpoints) ListenAndServe(ctx context.Context) error {
 	entryv1_pb.RegisterEntryServer(udsServer, e.APIServers.EntryServer)
 	svidv1_pb.RegisterSVIDServer(tcpServer, e.APIServers.SVIDServer)
 	svidv1_pb.RegisterSVIDServer(udsServer, e.APIServers.SVIDServer)
+	// Register Debug API only on uds server
+	debugv1_pb.RegisterDebugServer(udsServer, e.APIServers.DebugServer)
```

Comment:
Is there any sort of test that ensures that this is NOT available on the TCP server? I know our authorization middleware will prevent access even it is exposed but was just curious.

Reply:
:+1